internal/literals/literals.go

```diff
@@ -24,7 +26,31 @@ func randObfuscator() obfuscator {
 }
 
 // Obfuscate replace literals with obfuscated lambda functions
-func Obfuscate(files []*ast.File, info *types.Info, fset *token.FileSet, blacklist map[types.Object]struct{}) []*ast.File {
+func Obfuscate(files []*ast.File, info *types.Info, fset *token.FileSet, garbleLiterals string, blacklist map[types.Object]struct{}) []*ast.File {
+	var obfStrings, obfBytes, obfInts, obfFloats, obfBools bool
```

Comment:
Make this a single struct instead of multiple variables

Reply:
or a bitset, like:

```
type obfuscateLiterals uint32

const (
    obfuscateStrings obfuscateLiterals = 1 << iota
    obfuscateBytes
    [...]

    obfuscateAll obfuscateLiterals = math.MaxUint32
)

[...]

var obfLits obfuscateLiterals
if garbleLiterals == "all" {
    obfLits = obfuscateAll // don't allow "all,foo"
} else {
    for _, op := range [...] {
    switch op {
        case "strings":
            obfLits |= obfuscateStrings
        [...]
}

[...]

if obfLits&obfuscateStrings != 0 { ... }
```

Besides being easier to implement (once you define the iota), we could also have other groups or sub-sets. As a silly example:

```
obfuscateByteSequences = obfuscateStrings | obfuscateBytes
```

Reply:
Bitset would be even better